### PR TITLE
APPSRE-10723 set konflux image as default

### PR DESCRIPTION
**Pull request by fishi0x01**
Konflux image is already running in stage and prod.
Lets use the konflux image as default.

### CODE REVIEW

**Comment by maorfr** [comment]
the image is private, and github-mirror is an open source tool.

File: openshift/github-mirror.yaml [hunk 1]
```diff
@@ -137,9 +137,9 @@ objects:
 
 parameters:
 - name: IMAGE
-  value: quay.io/app-sre/github-mirror
+  value: quay.io/redhat-services-prod/app-sre-tenant/github-mirror-master/github-mirror-master
```
Comment (hemslo) [inline openshift/github-mirror.yaml:140]: Is there any way to grant our quay accounts access to this image or make it public like before?

Comment (fishi0x01) [reply openshift/github-mirror.yaml:140]: Fixed

Comment (hemslo) [reply openshift/github-mirror.yaml:140]: what's the difference between `redhat-user-workloads` and `redhat-services-prod`?

**Pull request closed without merging**